src/KeyDB.php

```diff
@@ -156,11 +156,11 @@ public function addKey(\ArrayAccess $keyMeta)
      *
      * @see https://github.com/theupdateframework/specification/blob/master/tuf-spec.md#4-document-formats
      */
-    public function getKey(string $keyId)
+    public function getKey(string $keyId):\ArrayObject
```

Comment:
IMHO the return type hint should be `\ArrayAccess`, since that's an interface, as opposed to a concrete class.

Reply:
We have other case in the project that are using `\ArrayObject` but none that are using `\ArrayAccess`.  So I would rather keep in the same.  Also `\ArrayObject` implements 3 other interfaces so it actually is more descriptive. We know will be an `\ArrayObject`